Element.js

```diff
@@ -257,19 +257,29 @@ Element.prototype = {
 	 * @returns {Promise.<void>}
 	 */
 	type: function (value) {
-		function getPostData(arrayValue) {
-			if (self.session.capabilities.isWebDriver) {
-				return { value: arrayValue.join('').split('') };
+		function getPostData(arrayValue, paramName) {
+			var obj = {};
+			obj[paramName] = null;
+			if (self.session.capabilities.isWebDriver &&
+				!self.session.capabilities.typeParameterCalledText) {
+				obj[paramName] = arrayValue.join('').split('');
+				return obj;
```

Comment:
Since you have an `else`, rather than returning here, why not just set `arrayValue = arrayValue.join('').split('')` here and let execution flow to the return at the end of the function?

Reply:
Gotcha! Fixed.